std/socket.d

```diff
@@ -3150,6 +3153,11 @@ public:
             from.setNameLen(nameLen);
             assert(from.addressFamily == _family);
             // if (!read) //connection closed
```

Comment:
I think you mean to delete these lines, so they're replaced with the lines added below.

Reply:
Done!